.github/workflows/ci-dockers.yml

```diff
@@ -123,6 +123,7 @@ jobs:
         timeout-minutes: 55
 
   build-docs:
+    # todo: this may not be needed
```

Comment:
are you planning to resolve this todo in this pr?

Reply:
dropped